src/utils/__snapshots__/PilotPromptCreator.test.ts.snap

```diff
@@ -0,0 +1,264 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PilotPromptCreator should create a prompt for an intent correctly 1`] = `
```

Comment:
one thing that I like about these kind of snapshots tests that they help me review the prompt without the code that generates it

Reply:
I'll write my comments on the snapshot but of course we'll need to update them on the creator

Reply:
This is really a good prompt. ✨